flow/controllers/base_controller.py

```diff
@@ -56,7 +58,18 @@ def __init__(self,
         self.delay = delay
 
         # longitudinal failsafe used by the vehicle
-        self.fail_safe = fail_safe
+        if isinstance(fail_safe, str):
+            self.fail_safe = [fail_safe]
+        elif isinstance(fail_safe, list) or fail_safe is None:
+            self.fail_safe = fail_safe
+        else:
+            print(
```

Comment:
we can replace print with a ValueError exception

Reply:
like this?
```
        try:
            if isinstance(fail_safe, str):
                self.fail_safe = [fail_safe]
            elif isinstance(fail_safe, list) or fail_safe is None:
                self.fail_safe = fail_safe
        except ValueError:
            print(
```